baseapp-wagtail/baseapp_wagtail/base/rest_framework/tests/test_api_sitemap.py

```
@@ -0,0 +1,46 @@
+        self.assertFalse(self._has_page(response.json(), new_page.id))
+
+    def _add_page(self, live=False):
+        new_page = StandardPage(title="My Page Child", slug="mypage-child", live=live)
+        self.page.add_child(instance=new_page)
+        return new_page
+
+    def _has_page(self, pages, page_id):
+        for page in pages:
+            if page["id"] == page_id:
+                return True
+        return False
```

Ruff

Use `return any(page["id"] == page_id for page in pages)` instead of `for` loop

Replace with `return any(page["id"] == page_id for page in pages)`

(SIM110)